ci/cluster-api/vsphere/templates/cluster.yaml

```diff
@@ -128,7 +80,11 @@ spec:
     - echo "127.0.0.1   localhost" >>/etc/hosts
     - echo "127.0.0.1   {{ ds.meta_data.hostname }}" >>/etc/hosts
     - echo "{{ ds.meta_data.hostname }}" >/etc/hostname
+    - ip addr add CONTROLVIP/32 dev eth0
     - ip link set eth0 mtu 1442
+    - ethtool -K eth0 tx-udp_tnl-segmentation off tx-udp_tnl-csum-segmentation off
+    # Antrea supports Geneve tunnel checksum offload, but throughput may drop sharply due to double encapsulation issues on overlay networks when it cannot correctly process checksum offloading.
```

Comment:
The capv testing environment is not in double encapsulation scenario. The checksum offload issue can happen also with some buggy NIC models.

This comment is a bit misleading, making people think that we need to disable checksum offloading for double encapsulation always.

Maybe we can use a comment like: Workaround tunnel checksum offloading issue in some problematic underlying network environments.

Reply:
Ok, comments updated.